apps/docs/pages/guide/create-commandkit.mdx

```diff
@@ -12,14 +14,28 @@ npm create commandkit@latest
 
 This will start the CLI in the current directory.
 
+### Development version
+
+Similar to the main package, you could try the `dev` or `next` versions of create-commandkit:
+
+<Callout type="warning">
+  This version is likely to have issues. Be careful!
```

Comment:
I think we should have some consistency with the [installation page](https://commandkit.js.org/guide/installation)

"The development version is likely to have bugs."